app/js/Features/Messages/MessageManager.js

```diff
@@ -13,11 +13,16 @@
  * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
  */
 let MessageManager
-const { ObjectId, getCollection } = require('../../mongodb')
+const { ObjectId, getCollection, waitForDb } = require('../../mongodb')
 const metrics = require('metrics-sharelatex')
 const logger = require('logger-sharelatex')
 
-const messagesCollectionPromise = getCollection('messages')
+let db
+waitForDb().then(async function () {
+  db = {
+    messages: await getCollection('messages')
+  }
+})
```

Comment:
You'll find me nitpicky, but how about doing this in [/app/js/mongodb.js](https://github.com/overleaf/chat/blob/master/app/js/mongodb.js) for all collections in one shot and exporting the `db` object?

Then, in modules, you simply import `db`:
```
const  { db } = require('../../mongodb')
...
db.messages.insertOne(...)
```

Reply:
Done an force-pushed along with the removal of the getCollection helper -- it's not used anymore: https://github.com/overleaf/chat/compare/d462c6d0745bd83fd3e00ff9e8cccb3de3795094..f68976a8c6523fbd64696b682746b980b28ba529